dependency_manager/src/edm_tool/edm.py

```diff
@@ -790,13 +790,15 @@ def write_cmake(cls, workspace: dict, checkout: list, dependencies: dict, out_fi
     @classmethod
     def check_github_key(cls) -> bool:
         """Checks if a public key is stored at github."""
-        valid = False
+        valid = True
         try:
             subprocess.run(["ssh", "-T", "git@github.com"],
                            stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True)
+        except subprocess.TimeoutExpired:
+            valid = False
         except subprocess.CalledProcessError as process_error:
             if process_error.returncode == 1:
-                valid = True
+                valid = False
```

Comment:
You changed the logic of this check. Indeed, a "successful" `ssh -T git@github.com` has an exit code of `1`.

I'm not sure whether a check for `1` can be a false positive though.

Reply:
Oh, got it, it shouldn't be False for 1